lib/src/test/resources/compileContracts.sh

```diff
@@ -8,16 +8,9 @@ readonly V1_SCARB_VERSION="0.4.0"
 readonly V2_SCARB_VERSION="0.7.0"
 readonly V2_6_SCARB_VERSION="2.6.0"
 
-# TODO: (#365) Use asdf action to install scarb instead
-echo "Installing scarb..."
-asdf plugin add scarb || true
-echo "Done!"
-
-
 echo "Compiling v1 contracts.."
 pushd "$(dirname "$0")" || exit 1
 pushd "$V1_CONTRACT_PATH" || exit 1
-asdf install scarb $V1_SCARB_VERSION || true
 asdf local scarb $V1_SCARB_VERSION || exit 1
```

Comment:
I wonder if we could use `asdf shell` instead? that way we would not create the .tool-versions in subdirectories, and still be able to use multiple versions across this script

Reply:
In order to use `asdf shell` in script we would have to source rc file in it.

Reply:
why?

Reply:
Because using `asdf shell` in a script without sourcing results in an error saying `Shell integration is not enabled. Please ensure you source asdf in your shell setup.` So we need to either set/source it.

Reply:
what is the effort to source this file, is it under the same path for both linux/macos and shells, or is the path different? Would it make sense to use paths to versions directly?